waku/waku_core/multiaddrstr.nim

```diff
@@ -0,0 +1,28 @@
+when (NimMajor, NimMinor) < (1, 4):
+  {.push raises: [Defect].}
+else:
+  {.push raises: [].}
+
+import
+    libp2p/[peerinfo, switch]
+
+import
+  ./peers
+
+
+
+proc constructMultiaddrStr*(wireaddr: MultiAddress, peerId: PeerId): string =
+  # Constructs a multiaddress with both wire address and p2p identity
+  $wireaddr & "/p2p/" & $peerId
```

Comment:
nitpick: I think is better to explicitly add `return` so that the code is clearer to new Nim devs. Applies elsewhere.

Reply:
Sure!